src/components/App/App.js

```diff
@@ -52,32 +82,60 @@ class App extends React.Component {
 					loadModel
 					confirmMatch={this.setPrediction}
 				/>
-				{this.props.prediction && (
-					<ConfirmationBox
-						item={this.props.prediction.index}
-						onYes={() => {
-							let id = getUserSlackID(
-								this.props.currentUser,
-								this.props.users
-							);
-							const name = labels[this.props.prediction.index][0];
-							sendSlackMessage(id, name, this.getStoreCode(name));
-							this.props.setPrediction(null);
-						}}
-						onNo={() => {
-							this.props.setPrediction(null);
-							this.props.setShowList(true);
-						}}>
-						<img src={this.props.prediction.img} alt="" />
-					</ConfirmationBox>
-				)}
-				{this.props.showList && <StoreListContainer />}
-				{this.props.slackUserFetchError && (
-					<ErrorMessage text="failed to fetch users" />
+				{this.props.prediction &&
+					this.props.storeList[this.props.prediction.id] && (
+						<ConfirmationBox
+							item={
+								this.props.storeList[this.props.prediction.id]
+							}
+							onYes={() => {
+								let id = getUserSlackID(
+									this.props.currentUser,
+									this.props.users
+								);
+								const name = this.props.storeList[
+									this.props.prediction.id
+								].name;
+								sendSlackMessage(
+									id,
+									name,
+									this.props.prediction.id
+								);
+								this.props.setPrediction(null);
+								this.showNotification('Reminder sent to Slack');
+							}}
+							onNo={() => {
+								this.props.setPrediction(null);
+								this.props.setShowList(true);
+							}}>
+							<img src={this.props.prediction.img} alt="" />
```

Comment:
Could we provide alternative text if the image couldn't be loaded? Maybe the item name?

e.g `alt={name}`